myapp/app/views/tasks/index.html.erb

```diff
@@ -7,6 +7,12 @@
                 new_task_path,
                 class: "btn btn-primary" %>
   </div>
+  <div>
+    <%= link_to "Sort by Created At ASC", tasks_path(sort_column: "created_at", sort_direction: "ASC") %>
+  </div>
+  <div>
+    <%= link_to "Sort by Created At DESC", tasks_path(sort_column: "created_at", sort_direction: "DESC") %>
+  </div>
```

Comment:
```suggestion
  <div>
    <%= link_to "Sort by Created At ASC", tasks_path(sort_column: 'created_at', sort_direction: 'ASC') %>
  </div>
  <div>
    <%= link_to "Sort by Created At DESC", tasks_path(sort_column: 'created_at', sort_direction: 'DESC') %>
  </div>
```

In Ruby, double quotation marks are often used for string interpolation, and single quotation marks are used otherwise.
In Rakuma, developers are writing code according to this `EnforcedStyle: single_quotes` conventions.

https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/StringLiterals

I know it is hard to get used to ruby syntax, so please introduce Linter!
This is the linter configuration used by our team.

https://github.com/Fablic/fablicop

Reply:
Though spent a lot of time, I think I successfully configured the CI workflow~